src/table/header-cell/styles.scss

```diff
@@ -17,6 +17,25 @@
   font-weight: awsui.$font-weight-heading-s;
   @include styles.font-smoothing;
   padding: awsui.$space-scaled-xxs awsui.$space-scaled-xs;
+
+  @include focus-visible.when-visible {
+    @include styles.focus-highlight(
+      (
+        'vertical': calc(-1 * #{awsui.$space-scaled-xxs}),
+        'horizontal': calc(-1 * #{awsui.$space-scaled-xxs}),
+      )
+    );
+  }
+  // stylelint-disable-next-line selector-combinator-disallowed-list
```

Comment:
The `body[data-awsui-focus-visible='true']` is required to only show focus outline when the keyboard input was detected. We don't document it elsewhere in the code but it could make sense to take a holistic look and maybe introduce a mixin utility.

Reply:
This block is written twice:
```
    @include styles.focus-highlight(
      (
        'vertical': calc(-1 * #{awsui.$space-scaled-xxs}),
        'horizontal': calc(-1 * #{awsui.$space-scaled-xxs}),
      )
    );
```

Reply:
Yes, but these are nested differently, I am not sure it is possible to de-duplicate.

Reply:
I think we could make a local mixin, but not a blocker.